geopy/geocoders/osm.py

```diff
@@ -248,6 +249,58 @@ def reverse(
             self._call_geocoder(url, timeout=timeout), exactly_one
         )
 
+    def lookup(
+            self,
```

Comment:
indentation. (if pep8 really told you to do this, then fine, but that would mean pep8 is worse than i thought...)